static/js/apps/eval_rig/data_store.tsx

```diff
@@ -91,3 +98,29 @@ export async function saveToSheet(
     [DC_STAT_FEEDBACK_COL]: response.dcStat,
   });
 }
+
+/**
+ * Gets whether records for a call exists
+ * @param sheetId the sheet id
+ * @param queryId the query id
+ * @param callId the call id
+ */
+export async function getCallExistence(
+  sheetId: string,
+  queryId: string,
+  callId: string
+): Promise<boolean> {
+  // Define the document reference
+  const collectionRef = collection(
+    db,
+    "sheets",
+    sheetId,
+    "queries",
+    queryId,
+    "calls",
+    callId,
+    "responses"
+  );
+  const snapshot = await getCountFromServer(collectionRef);
+  return !!snapshot.data().count;
```

Comment:
similar to this, but use path (db, "sheets", sheetId, "queries", queryId, "calls") would get the count of calls under a query?

Reply:
updated